include/rcpputils/filesystem_helper.hpp

```diff
@@ -224,7 +223,7 @@ class path
   bool is_absolute() const
   {
     return path_.size() > 0 &&
-           (path_.compare(0, 1, std::string(1, kPreferredSeparator)) == 0 ||
+           (path_[0] == kPreferredSeparator ||
```

Comment:
just a confirmation, this is only for performance improvement?

Reply:
Yep, no need to create a new string object to compare one character :).